### DISCUSSION TITLE
LaTeX render in GitHub mobile app

### DISCUSSION THREAD

**Discussion by andr-ll** [community/community#41608] [Projects and Issues]
Hello!

I've been writing some documentation for my repo and found out that mobile version of app does not show valid output for LaTeX syntax.

Mobile app:
![image](https://user-images.githubusercontent.com/72939898/206879125-ae5bd384-ba57-46b2-a267-06d46632b97b.jpeg)

GitHub web:
![image](https://user-images.githubusercontent.com/72939898/206879132-d9abbf88-7371-4fb9-aae2-71a46b655a34.jpeg)

The source file is [available here](https://github.com/andr-ll/nuti/blob/master/docs/logger.md).

Seems like this is a small issue.

Thanks in advance :)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.92.0 (85285908)
Device: iPhone Xs (iOS 16.1.2)
</details>

**Comment by andr-ll** [comment]
Same discussion with some comments:

https://github.com/orgs/community/discussions/41372

**Reply by markcst** [reply]
It's almost 2025 and Microsoft still hasn't done this crap?
And move along, you just need to implement Latex rendering in an app, not go to the moon!

**Reply by GioByte10** [reply]
Such a fresh comment, thought I would be the only one.
